### ISSUE TITLE
admin functionality

### ISSUE THREAD

**Issue by mcbouslog** [zooniverse/pfe-lab#34]
- [ ] review and plan Admin functionality
- [ ] implement Admin functionality (to be updated with more details)
- [ ] footer checkbox
- [ ] page border

**Comment by amyrebecca** [comment]
Is there anything left to do on this once PR #35 is merged?

**Comment by mcbouslog** [comment]
Nope, after that PR this issue can be closed.

Eventually we can add an admin page like on PFE where you can search and edit users and projects, and add orgs, but that can be its own issue, and I think a low priority.
